### PR TITLE
Add PLEP 4 on licensing of PlasmaPy repositories

### PR DESCRIPTION
**Pull request by namurphy**
The purpose of this PlasmaPy Enhancement Proposal is to add an explicit patent grant to PlasmaPy's BSD 3-clause license to protect the rights of users against software patents, and to have creative works covered by Creative Commons licenses.

The associated pull request for the main repository is: https://github.com/PlasmaPy/PlasmaPy/pull/114
Additional discussion is at: https://github.com/PlasmaPy/PlasmaPy-Tutorials/issues/1

### CODE REVIEW

File: PLEP-0004.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+[proprietary](https://en.wikipedia.org/wiki/Proprietary_software)
+licenses.  The choice of this license was to follow the established
+practices of projects such as [Astropy](http://www.astropy.org/).
+
+Most modern software licenses include language that protects against
+software patents.  The [BSD 3-clause
+license](https://opensource.org/licenses/BSD-3-Clause) does not
+explicitly include such language.  This oversight leaves potential for
+a contributor to patent their contribution, and then prohibit use of
+thier contribution by the PlasmaPy community.  Plasma physics has a
```
Comment (StanczakDominik) [inline PLEP-0004.md]: their contribution*

File: PLEP-0004.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+- PlasmaPy could use the [BSD 3-clause
+  license](https://opensource.org/licenses/BSD-3-Clause) without the
+  patent grant; however this would remove explicit protections against
+  software patents.
+
+- Another alternative would be to use the [Apache License, Version
+  2.0](https://opensource.org/licenses/Apache-2.0) which includes
+  specific protections against patents.  However, [the naming of the
+  Apache license constitutes cultural
+  appropriation](https://github.com/Quick/Quick/issues/660) which
```
Comment (StanczakDominik) [inline PLEP-0004.md]: Whoa, never thought about this issue.

File: PLEP-0004.md [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+4.0) license](https://creativecommons.org/licenses/by-sa/4.0/)] is
+desirable, then a PlasmaPy repository may be licensed under one or
+more compatible copyleft licenses.  When PlasmaPy adopts or
+incorporates an existing body of creative works released under an
+[Open Source Initiative (OSI) approved
+license](https://opensource.org/licenses/category) or a [Creative
+Commons license](https://creativecommons.org/licenses/), then that
+body of work and its derivatives may continue to use that license.
+PlasmaPy's Coordinating Committee may make exceptions to this policy
+when there extenuating circumstances.
```
Comment (SolarDrew) [inline PLEP-0004.md]: when there are extenuating circumstances.

**Comment by namurphy** [comment]
Coordinating Committee members - please vote on PLEP 4! Any additional suggestions from the community are still appreciated.

I vote in favor. slightly_smiling_face +1

**Comment by StanczakDominik** [comment]
:+1: :smiley:

**Comment by tulasinandan** [comment]
I agree with the suggestions made by The Master of Licenses! 

I vote in favor! :+1:

**Comment by namurphy** [comment]
Over 2/3 of the Coordinating Committee has voted in favor, so this passes too!

**Pull request merged**